contracts/VTokenInterfaces.sol

```diff
@@ -353,4 +360,11 @@ abstract contract VTokenInterface is VTokenStorage {
     function setInterestRateModel(InterestRateModel newInterestRateModel) external virtual;
 
     function addReserves(uint256 addAmount) external virtual;
+
+    /**
+     * @dev This empty reserved space is put in place to allow future versions to add new
+     * variables without shifting down storage in the inheritance chain.
+     * See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
+     */
+    uint256[50] private __gap;
```

Comment:
Same here. As `VTokenStorage` is our base storage contract that we inherit from, shouldn't we only have the gap in the `VTokenStorage` interface, rather than both interfaces?

Reply:
Same comment as above.

Reply:
Same comment as above